[2021-09-27 21:48:51,998] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.say_hello 2021-09-27T20:40:00+00:00 [queued]>
[2021-09-27 21:48:52,062] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: my_dag.say_hello 2021-09-27T20:40:00+00:00 [queued]>
[2021-09-27 21:48:52,064] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 21:48:52,065] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2021-09-27 21:48:52,069] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2021-09-27 21:48:52,091] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): say_hello> on 2021-09-27T20:40:00+00:00
[2021-09-27 21:48:52,106] {standard_task_runner.py:52} INFO - Started process 25361 to run task
[2021-09-27 21:48:52,162] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'say_hello', '2021-09-27T20:40:00+00:00', '--job-id', '96', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpl5z1mviv', '--error-file', '/tmp/tmpk_fmoc1r']
[2021-09-27 21:48:52,175] {standard_task_runner.py:77} INFO - Job 96: Subtask say_hello
[2021-09-27 21:48:52,442] {logging_mixin.py:109} INFO - Running <TaskInstance: my_dag.say_hello 2021-09-27T20:40:00+00:00 [running]> on host 7c1d2d2abdb5
[2021-09-27 21:48:52,716] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Veerawat
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=say_hello
AIRFLOW_CTX_EXECUTION_DATE=2021-09-27T20:40:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-09-27T20:40:00+00:00
[2021-09-27 21:48:52,721] {my_dag.py:11} INFO - Hello, SAKSIAM
[2021-09-27 21:48:52,724] {python.py:151} INFO - Done. Returned value was: None
[2021-09-27 21:48:52,786] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=say_hello, execution_date=20210927T204000, start_date=20210927T214852, end_date=20210927T214852
[2021-09-27 21:48:52,966] {local_task_job.py:151} INFO - Task exited with return code 0
[2021-09-27 21:48:53,110] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
